src/components/header.component.vue

```diff
@@ -0,0 +1,395 @@
+<template>
+  <div class="HeaderWrapper">
+    <div class="HeaderBox">
+      <span class="LogoWrapper" v-bind:class="isShowCategory ? 'Active' : ''">
```

Comment:
v-bind는 축약문자로 ":"를 쓸 수 있습니다. 마찬가지로 v-on은 "@"로 축약됩니다.
가독성 측면에서 더 유리하니까 축약문자를 적극 활용하면 좋을 것 같아요
```suggestion
      <span class="logo-wrapper" :class="isShowCategory ? 'Active' : ''">
```

Reply:
그리고 Vue 공식문서에서는 클래스에 객체를 바인딩하는 방법이 소개됩니다.
삼항 연산자를 쓸 경우, 클래스를 여러개 바인딩해야 하는 상황에서 가독성이 매우 떨어지므로 객체 방식을 채택하면 좋을 것 같습니다.
```suggestion
      <span class="logo-wrapper" :class="{'active': isShowCategory}">
```